packages/calcite-components/src/components/carousel-item/carousel-item.scss

```diff
@@ -0,0 +1,13 @@
+:host {
+  display: none;
```

Comment:
Instead of display: none on the host could we do it on an internal container so we're not susceptible to any user styles overriding this? Then the default display could be flex or block.